parity.md

```
@@ -0,0 +1,52 @@
+# RiptOPL vs. wOPL Feature Parity & Implementation Review
+
+This document provides a highly critical technical comparison between **RiptOPL** (our current build) and **wOPL** (the `wOPL-base` branch). Both builds are forks of Open PS2 Loader with substantial additions; however, their approaches to implementing similar features differ significantly in execution, architecture, and PS2 hardware safety.
+
+## 1. Theme Engine: Coverflow Implementation
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)